Bukkit/src/main/java/me/fixeddev/commandflow/bukkit/part/GameModePart.java

```diff
@@ -69,11 +66,16 @@ public GameModePart(String name) {
         }
 
         String possibleGameModeName = next.toUpperCase();
+        List<String> suggestions = new ArrayList<>();
+
+        for (GameMode gameMode : GameMode.values()) {
```

Comment:
GameMode array returned by `GameMode.values()` can be cached, `values()` creates a new array every time it is called